cmd/tag-set.go

```diff
@@ -0,0 +1,122 @@
+/*
+ * MinIO Client (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"errors"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/console"
+)
+
+var tagSetCmd = cli.Command{
+	Name:   "set",
+	Usage:  "set/configure tags for an object",
```

Comment:
to be consistent with other mc commands like mc policy/event, we could use add, remove and list subcommands instead of set, remove and show

Reply:
I will make the change but to be sure, for this functionality the existing tags are completely replaced with the new set of tags.